<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Waagle</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>191</width>
      <height>80</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="load">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>41</width>
        <height>9</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="logo_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>41</width>
        <height>9</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>0</y>
        <width>80</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="search">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>80</width>
        <height>15</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>search</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="logo">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>-20</y>
        <width>181</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>120</width>
        <height>80</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="searchscroll">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>118</width>
         <height>78</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="topage1_1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>80</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="topage2_1">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>80</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="topage3_1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>80</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="topage1_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>80</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="topage2_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>80</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="topage3_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>80</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>120</width>
        <height>80</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="searchscroll_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>118</width>
         <height>78</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="logo_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>181</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>70</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="search_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>70</y>
        <width>80</width>
        <height>15</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>search</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="topage1_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>80</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="topage2_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>80</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="topage3_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>80</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea_3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>120</width>
        <height>80</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="searchscroll_7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>118</width>
         <height>78</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="logo_3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>181</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="search_3">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>90</y>
        <width>80</width>
        <height>15</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>search</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>14</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
